interfaces/ldconfig/spec.go

```diff
@@ -61,10 +63,18 @@ func (spec *Specification) AddLibDirs(dirs []string) error {
 	return nil
 }
 
+func (spec *Specification) AddPlug(plug string) {
```

Comment:
this doesn't need to be public and done by the interfaces, we can do it for them in:

https://github.com/canonical/snapd/blob/71aca8fc8d61734208beb93ee666785b6c401a21/interfaces/ldconfig/spec.go#L114

no?

Reply:
Right, I've changed this now: https://github.com/canonical/snapd/pull/15063/commits/a5b040c4af66118aa766b9e1f29d976839f71254